default.json

```diff
@@ -73,53 +63,11 @@
       "groupName": "Size limit packages",
       "groupSlug": "size-limit"
     },
-    {
-      "description": "Group apollo packages",
-      "matchPackagePrefixes": ["@apollo"],
-      "groupName": "Apollo packages",
-      "groupSlug": "apollo"
-    },
     {
       "description": "Skip config updates (breaking change at 3.3.8)",
       "matchPackageNames": ["config"],
       "allowedVersions": "<=3.3.7"
     },
-    {
-      "description": "Require stability of NPM dependencies which are not managed by Side Inc., Google or another trusted organization (since they can be unpublished)",
```

Comment:
So we replaced this with confidence? We might want to keep skipping of excludes for `@side/*` deps since renovate won't have enough downloads to determine a clear confidence (if at all it will be low)

Reply:
I can bring that back. I removed it because I didn't see it in the Renovate configuration anywhere. You know where that's located?

Reply:
I added `@side` above as the only thing we exclude. Everything else is using the confidence score instead.